docs/get-started/ai-machine-learning.md

```diff
@@ -0,0 +1,112 @@
+* **AutoML**. The system self-tunes, including seasonality detection, minimizing user intervention for a simpler experience.
+* **User context**. Users set alert sensitivity and incident thresholds, adding context to anomaly detection to mitigate noise.
+* **One-click playbook assignment**. Monitors seamlessly link to Automation Service playbooks, expediting response without manual intervention.
+* **Auto-diagnosis and recovery**. Sumo Logic Automation Service automates diagnosis and resolution, closing the loop from alert to recovery.
+
+#### Automated playbooks
+
+With our [Automated playbooks](/docs/alerts/monitors/use-playbooks-with-monitors), you can set up a predefined set of actions and conditional statements that respond to an events like security incidents by running an automated workflow. Configuration is easy - browse our 500+ existing playbooks in the Automation Service App Central, then choose and/or customize it.
+
+### Apps
+
+Sumo Logic offers integrations with various AI-driven platforms to analyze large volumes of data, including:
+
+* [Google Cloud Vertex AI](/docs/integrations/google/cloud-vertex-ai)
+* [OpenAI ChatGPT](/docs/platform-services/automation-service/app-central/integrations/openai-chatgpt)
+* [Vectra AI](/docs/platform-services/automation-service/app-central/integrations/vectra)
+* [Microsoft Sentinel](/docs/platform-services/automation-service/app-central/integrations/microsoft-sentinel)
+* [Darktrace](/docs/platform-services/automation-service/app-central/integrations/darktrace)
+* [Criminal IP](/docs/platform-services/automation-service/app-central/integrations/criminal-ip)
+* [Arcanna](/docs/platform-services/automation-service/app-central/integrations/arcanna)
+
+<!-- Removed - not in Kanso
+### Root Cause Explorer
+
+Accelerates troubleshooting by surfacing events of interest correlated with application incidents. [Learn more](/docs/observability/root-cause-explorer)
+-->
+
+## Security
+
+### Cloud SIEM
+
+Sumo Logic's Cloud SIEM leverages AI-driven rules for security management, including:
+
+* [Insight Trainer](/docs/cse/rules/insight-trainer). Utilizes ML and AI to deliver outcome-based recommendations, reducing false positives without compromising detection value.
+* [First-seen rule](/docs/cse/rules/write-first-seen-rule). Identifies novel threats based on first occurrences.
+* [Outlier rule](/docs/cse/rules/write-outlier-rule/). Detects abnormal behavior indicating potential security breaches.
+
+### Global Intelligence
+
+Our Global Intelligence Service apps provide security teams with valuable real-time security intelligence to scale detection, prioritization, investigation, and workflow to prevent potentially harmful service configurations that could lead to a costly data breach.
+
+* [Global Intelligence Service](/docs/integrations/global-intelligence)
```

Comment:
```suggestion
Our Global Intelligence Service apps provide security teams with valuable real-time security intelligence to scale detection, prioritization, investigation, and workflow to prevent potentially harmful service configurations that could lead to a costly data breach. [Learn more](/docs/integrations/global-intelligence).
```